python/ray/_private/runtime_env/uv.py

```diff
@@ -79,14 +79,22 @@ async def _install_uv(
         virtualenv_path = virtualenv_utils.get_virtualenv_path(path)
         python = virtualenv_utils.get_virtualenv_python(path)
 
```

Comment:
After 5/N the high level logic should be

```
if uv_version is not None:
  pip install --ignore-installed uv=xxxx
else:
  if uv is installed:
    do nothing
  else:
    installed the latest
```

?

Reply:
Sounds good, btw did some quick research, seems `--force-reinstall` is preferred.
Eg: https://stackoverflow.com/questions/51913361/difference-between-pip-install-options-ignore-installed-and-force-reinstall

Reply:
Updated.